tomography_preprocessing/tilt_series_alignment/aretomo.py

```diff
@@ -0,0 +1,48 @@
+from pathlib import Path
+from typing import Optional
+
+import typer
+from lil_aretomo.aretomo import run_aretomo_alignment
+
+from ._cli import cli
+from .. import utils
+
+ARETOMO_ALIGNMENT_COMMAND_NAME = 'AreTomo'
+
+@cli.command(name=ARETOMO_ALIGNMENT_COMMAND_NAME)
+def aretomo_function(
+        tilt_series_star_file: Path = typer.Option(...),
+        output_directory: Path = typer.Option(...),
+        exe: Path = typer.Option(...),	
+        local_align: Optional[bool] = typer.Option(False),
+        target_pixel_size: Optional[float] = typer.Option(10),
+        patch_in_x: Optional[float] = typer.Option(5),
+        patch_in_y: Optional[float] = typer.Option(4),
+        correct_tilt_angle_offset: Optional[bool] = typer.Option(False),
+        tomogram_name: Optional[str] = typer.Option(None)
```

Comment:
just checking, are these all optional? Do things run correctly if any of these are absent?

Reply:
Yep! AT will automatically align using global alignment, local alignment is an option. Default pixel size is 10A/px, the user can change that if they like but don't have to. Patches only come into play if local align is selected. Correct tile angle offset is off by default. Tomo name is only for individual tomos